website_portal/__openerp__.py

```diff
@@ -5,10 +5,16 @@
     'summary': 'Account Management Frontend for your Customers',
     'version': '8.0.1.0.0',
     'license': 'LGPL-3',
-    'author': 'Odoo SA, MONK Software, Antiun Ingeniería S.L.',
+    'author': (
+        'Odoo SA'
+        ', MONK Software'
+        ', Antiun Ingeniería S.L.'
+        ', Odoo Community Association (OCA)'
+    ),
     'website': (
-        'https://www.odoo.com/, http://www.monksoftware.it, '
-        'https://github.com/OCA/website'
+        'https://www.odoo.com/'
```

Comment:
Sorry for not noticing before, but this URL should be unique and parseable or pypi upload will fail. We are right now putting the repository URL

Reply:
Sorry, I'm not clear on this. What do you mean by 'unique'? We can't have the same URL in multiple addons? Or it only counts for the first URL?

Reply:
You have to leave only one URL, not 3.